web/app.js

```diff
@@ -323,6 +325,9 @@ var PDFViewerApplication = {
     });
     this.pdfSidebar.onToggled = this.forceRendering.bind(this);
 
+    this.pageNumberInput = document.getElementById('pageNumber');
```

Comment:
The `getElementById` calls in this PR probably need to be moved to the viewer configuration part after #7209.